src/backend/InvenTree/order/api.py

```diff
@@ -1617,6 +1659,30 @@ def item_link(self, item):
             path(
                 '<int:pk>/',
                 include([
+                    path(
+                        'can_approve/',
+                        PurchaseOrderApprovalAllowed.as_view(),
+                        name='api-po-approval-allowed',
+                    ),
+                    path(
+                        'request_approval/',
+                        PurchaseOrderRequestApproval.as_view(),
+                        name='api-po-req-approval',
+                    ),
+                    path(
+                        'reject/', PurchaseOrderReject.as_view(), name='api-po-reject'
+                    ),
+                    path('ready/', PurchaseOrderReady.as_view(), name='api-po-ready'),
+                    path(
+                        'can_issue/',
+                        PurchaseOrderIssueAllowed.as_view(),
+                        name='api-po-issue-allowed',
+                    ),
+                    path(
+                        'pending/',
+                        PurchaseOrderPending.as_view(),
+                        name='api-po-pending',
+                    ),
```

Comment:
Why are we adding this many API endpoints for this? At least `can_approve`, `can_issue`, `pending` could be either merged or just added to the PO detail endpoint

Reply:
Pending is a state transition like others. But if you know a better way to assert if a user is able to approve/issue an order through other means, I'm all ears :)

Reply:
then it should be renamed - the name `pending` is not communicating that it is changing something, a verb  like `reject` or an action like `request_approval` would be better

Reply:
Hmm, yeah. I named it for its state, but I've later called it "recall" on buttons. So maybe recall might work better than pending. Rejecting uses the same endpoint, but from its own backend. What do you think about changing it to "Recall", "retract" or similar?

Reply:
recall or retract would both make sense